build/push-ci-build.sh

```diff
@@ -32,5 +32,11 @@ KUBE_GCS_LATEST_CONTENTS=${LATEST}
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "$KUBE_ROOT/build/common.sh"
 
-kube::release::gcs::release
-kube::release::gcs::publish_latest
+MAX_ATTEMPTS=3
+attempt=0
+while [[ ${attempt} -lt ${MAX_ATTEMPTS} ]]; do
+  kube::release::gcs::release && kube::release::gcs::publish_latest && break || true
```

Comment:
We're not running Jenkins with "yes | build" anymore, I don't think, are we?

If so, won't this hit https://github.com/GoogleCloudPlatform/kubernetes/blob/74b78d537df6dee6842573f2b78ea28dbdf87ade/build/common.sh#L910 ?